src/client/src/apps/SimpleLauncher/SimpleLauncher.tsx

```diff
@@ -82,7 +82,7 @@ export const SimpleLauncher: React.FC = () => {
             <PricingServiceProvider value={pricingService}>
               <ReferenceDataProvider value={referenceData}>
                 <PlatformProvider value={platform}>
-                  <Helmet title={getAppName('Reactive Launcher')} />
+                  <Helmet title={getAppName('Reactive Launcher™')} />
```

Comment:
Remove this one is only the phrase "Reactive Trader" has a trademark 😄

Reply:
Haha my bad, got rid of the TM on that line. Should be good now.